packages/material-ui-lab/src/SliderStyled/SliderStyled.js

```diff
@@ -303,6 +303,10 @@ const Slider = React.forwardRef(function Slider(inputProps, ref) {
   );
 });
 
+Slider.options = {
```

Comment:
Maybe to signal its internal?

```suggestion
Slider.__options = {
```

Reply:
I've used this name as that was the name used by the `withStyles`, so that we would persist the same API. But now that it is visible it may make sense to rename it.. Should we rename it in both components so it is consistent, or should we for now just rename this one?

Reply:
Oh, I see what you mean, yeah ok. We can keep it like this, simpler to keep backward compatibility for the docs generation and maybe the tests too.